src/pages/Home.tsx

```diff
@@ -1,9 +1,44 @@
-function Home() {
+import React from 'react';
+import compactImage from '../assets/compact.jpg';
+import sportCarImage from '../assets/sport-car.jpg';
+import welcabImage from '../assets/welcab.jpg';
+import vanImage from '../assets/van.jpg';
+import volvoImage from '../assets/volvo.jpg';
+import teslaImage from '../assets/tesla.jpg';
+import rvImage from '../assets/rv.jpg';
 
-    return (
-      <div className='home'>
+// Define a type for the car model
+type CarModel = {
+  name: string;
+  image: string;
+};
+
+const carModels: CarModel[] = [
+  { name: 'Compact', image: compactImage },
+  { name: 'Sport Car', image: sportCarImage },
+  { name: 'Welcab', image: welcabImage },
+  { name: 'Van', image: vanImage },
+  { name: 'Volvo', image: volvoImage },
+  { name: 'Tesla', image: teslaImage },
+  { name: 'RV', image: rvImage }
+];
+
+const Home: React.FC = () => {
+  return (
+    <div className="container mx-auto px-4">
+      <h2 className="text-2xl font-bold text-gray-800 my-8">Car models you might like</h2>
```

Comment:
For the landing page, we have some changes, we will have two sections, the first section is car brands, and the section is car types, the example as below:
`<h2> Car brands you might like</h2>
<div/>
<h2> Car types you might like </h2>
<div/>
`
the modes will be shown on the next page regarding the user has chosen which brand or type.

Reply:
BTW, the image urls are stored in the backend, but you can keep this way for our demo.
we will discuss it in the meeting on Wednesday.
Thanks!